app/templates/issues.hbs

```diff
@@ -1,17 +1,20 @@
 <div class="help-wanted-links">
   <a href="/">{{es-button label="All Issues" isTiny=true}}</a>
-  {{#each navLinks.navLinks as |link|}}
+  {{#each this.navLinks.navLinks as |link|}}
     <a href={{link.href}}>{{es-button label=link.name isTiny=true}}</a>
   {{/each}}
 </div>
-<form {{action "searchByWildcard" queryInput on="submit"}} class="search-box">
-  {{input value=queryInput}} <button type="submit">Search</button>
-  {{#if label}}
-    <span class="current-filter-label">{{label-with-emoji label}} <span {{action (mut label) null}}>x</span></span>
+<form {{action "searchByWildcard" this.queryInput on="submit"}} class="search-box">
+  {{input value=this.queryInput}} <button type="submit">Search</button>
+  {{#if this.label}}
+    <span class="current-filter-label">
+      {{label-with-emoji this.label}}
+      <button class="github-label-button" {{action (mut this.label) null}}>x</button>
```

Comment:
In `filter-test` (line 21) this was previously a `span` tag instead of a `button` and the test is still looking for a `span` tag.